### ISSUE TITLE
Quotes in XML cause highlight issue

### ISSUE THREAD

**Issue by dnicolson** [sbarex/SourceCodeSyntaxHighlight#229]
The following XML:
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Key1</key>
	<string>Quote start &quot;</string>
	<key>Key2</key>
	<string>Quote end &quot;</string>
	<key>Key3</key>
	<string>Test</string>
</dict>
</plist>
```

Breaks the syntax highlighting between the use of quotes:
<img width="682" alt="Syntax Highlight" src="https://user-images.githubusercontent.com/2276355/219962773-d778ca10-260a-4926-8a5c-7c25d732d191.png">

**Comment by sbarex** [comment]
Thanks, I've opened an issue in the [`Hightlight`](https://gitlab.com/saalen/highlight/-/issues/226) render engine.

**Comment by sbarex** [comment]
Fixed on release 2.1.18.
